src/Sylius/Bundle/ShopBundle/Resources/views/Email/contactRequest.html.twig

```diff
@@ -1,18 +1,10 @@
+{% deprecated 'The "@SyliusShop/Email/contactRequest.html.twig" template is deprecated since Sylius 1.12 and will be removed in Sylius 2.0, use "@SyliusCore/Email/contactRequest.html.twig" instead.' %}
```

Comment:
I'm not really sure about it, but as I see it we have 2 same templates used now and to keep the BC I leaved both of them and marked the one from ShopBundle as deprecated